front-end/src/hooks/useFetchUnits.js

```diff
@@ -12,14 +13,20 @@ const useFetchUnits = (isEdit, searchUnit, selectedLevel) => {
       try {
         if (!isEdit) {
           setLoading(true);
-          const units = await getUnitsByLevel(selectedLevel);
+          const units =
```

Comment:
Why you use `getUnitsByLevel` from question service in materials?

Reply:
I wrote a condition to get a request for the materials in my component, and i get request for materials only